core/kazoo_number_manager/src/kazoo_number_manager_maintenance.erl

```diff
@@ -176,7 +169,7 @@ refresh_numbers_dbs() ->
 -spec refresh_numbers_dbs(ne_binaries(), non_neg_integer()) -> 'ok'.
 refresh_numbers_dbs([], _) -> 'ok';
 refresh_numbers_dbs([NumberDb|NumberDbs], Total) ->
-    ?LOG("(~p/~p) updating number db ~s", [length(NumberDbs) + 1, Total, NumberDb]),
+    ?SUP_LOG_DEBUG("(~p/~p) updating number db ~s", [length(NumberDbs) + 1, Total, NumberDb]),
```

Comment:
Can you update `refresh_numbers_dbs()`:

```erlang
    NumberDbs = knm_util:get_all_number_dbs(),
    refresh_numbers_dbs(NumberDbs, length(NumberDbs)).
```

There's no `system_numbers` classification anyway :)

Reply:
done